docs/apt.md

```diff
@@ -0,0 +1,48 @@
+# apt BASH in the Command Line!
+
+## Summary 
+
+<p> What is the apt BASH command? </p>
+
+<p> I am so glad you asked! </p>
+
+<p> Used to install software packages through the command line, apt has many applications that make it exceptionally helpful for unpacking updates and new software. </p>
+
+<p> In order to use this command, you must have sudo/super-user privileges on your server or computer. If you are using your personal computer, you should be able to use this command without getting sudo permissions. </p>
+
+## Basic command structure
+For all descriptions below, the dollar sign indicates that BASH command prompt.
+
+$ sudo apt [argument]
+
+Note: check your spelling, use tab completion  
+
+## Possible Arguments
+
+### `update`
+Updates package lists from repositories
+<p> Example </p>
+ <p> $ sudo apt update </p>
```

Comment:
It's best formatting-wise to put full commands like the one on line 25 between [back ticks ](https://en.wikipedia.org/wiki/Backtick)which will create the code formatting. Like this: $ `sudo apt update`